plan/plan.go

```diff
@@ -271,14 +271,12 @@ type Rel interface {
 	// result should be 3 columns consisting of the 5th, 2nd and 1st
 	// output columns from the underlying relation.
 	OutputMapping() []int32
-	// Remap utilizes the OutputMapping to construct the expected result
-	// record type from the output RecordType of the underlying Relation.
-	// If the output mapping is nil, then this just returns the input
-	// type.
-	Remap(types.StructType) types.StructType
-	// RecordType returns the output record type of the underlying relation
-	// as a struct type.
-	RecordType() types.StructType
+	// DirectOutputSchema returns the output record type of the underlying
+	// relation as a struct type.
+	DirectOutputSchema() types.StructType
```

Comment:
Should we export this? I'm kind of inclined not to. What's your thoughts?

Reply:
It does appear that all uses are internal to the package with two instances of it being used to handle an "and Remap" operation that will be obsoleted.  Also, it is easier to later expose this if it turns out to be needed than hide it later.